src/apple.rs

```diff
@@ -3,13 +3,25 @@ use async_recursion::async_recursion;
 use chrono::Utc;
 use serde::{Deserialize, Serialize};
 use hyper::{body, Body, Client, Request};
+use hyper_tls::HttpsConnector;
 
 //https://developer.apple.com/documentation/appstorereceipts/status
 const APPLE_STATUS_CODE_TEST: i32 = 21007;
+const APPLE_PROD_VERIFY_RECEIPT: &str = "https://buy.itunes.apple.com";
+const APPLE_TEST_VERIFY_RECEIPT: &str = "https://sandbox.itunes.apple.com";
 
-pub struct AppleUrls {
-    pub production: String,
-    pub sandbox: String,
+pub struct AppleUrls<'a> {
+    pub production: &'a str,
+    pub sandbox: &'a str,
```

Comment:
Changed these to refs to avoid allocations because our exposed apple_response() call will take the consts as default values so users don't have to worry about redefining something that isn't going to change. This comes with a little refactoring in the tests

Reply:
make sense